configuration.rst

```diff
@@ -376,6 +376,19 @@ Configuration parameters are very common in Symfony applications. Some packages
 even define their own parameters (e.g. when installing the translation package,
 a new ``locale`` parameter is added to the ``config/services.yaml`` file).
 
+.. tip::
+
+    When working with :ref:`Compiler Passes </service_container/compiler_passes>`,
+    you have the possibility to declare parameters that will only be available
+    during the container compilation. These parameters will be automatically removed
+    from the container once its build is done. To declare parameters only
+    available at compile-time, pre-prend their name with a dot ``.`` (for example,
```

Comment:
Is "pre-prend" correct?

Reply:
No, it's prepend or prefix. But Javier secretly fixed this already in the main branch by rewording this tip https://github.com/symfony/symfony-docs/commit/48d710e9b0866bf66eaf997de55f4e1fd6560b8e

Reply:
Thanks for the information!